src/python/evaluation/support_functions.py

```diff
@@ -0,0 +1,22 @@
+import os
+from pathlib import Path
+from typing import NoReturn, Union
+
+from openpyxl import load_workbook
+
+
+def create_folder(directory: Union[str, Path]) -> NoReturn:
+    try:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+    except OSError:
+        pass
+
+
+def remove_sheet(path: str, name: str) -> NoReturn:
```

Comment:
left `workbook_path` as an argument that is passed to the function because we still need it to save changes after removing the sheet, the rest is resolved